include/JSystem/J2D/J2DPane.h

```diff
@@ -115,7 +116,7 @@ struct J2DPane {
 	J2DPane* getFirstChildPane();
 	J2DPane* getNextChildPane();
 	J2DPane* getParentPane();
-	JSUTree<J2DPane>* getPaneTree();
+	JSUTree<J2DPane>* getPaneTree() { return &m_tree; }
```

Comment:
definitely resolve this weak function in the future

Reply:
I assume this is actually about `J2DPane::getTagName` which is currently global when it should be weak - will need to revisit with more files decomp'd, agreed.